docs/configuration.md

```diff
@@ -137,14 +137,15 @@ Aside from the comments in YAML file itself the fields are described below.
 |                                                      | `min_skeleton_size`                     | int                                                | `10`                        | The minimum number of pixels a skeleton should be for statistics to be calculated on it. Anything smaller than this is dropped but grain statistics are retained.                                                                                                                                                                                                                                                                         |
 | `disordered_tracing` <br> ∟ `mask_smoothing_params`  | `gaussian_sigma`                        | float                                              | `2`                         | Amount of smoothing by a gaussian kernel. This will compete with `dilation_iteration` to see which changes the grain mask least, ensuring quality over different scan sizes.                                                                                                                                                                                                                                                              |
-|                                                      | `dialtion_iterations`                   | int                                                | `2`                         | The the number of dilations to perform to smooth. This will compete with `gaussian_sigma` to see which changes the grain mask least, ensuring quality over different scan sizes.                                                                                                                                                                                                                                                          |
+|                                                      | `dilation_iterations`                   | int                                                | `2`                         | The the number of dilations to perform to smooth. This will compete with `gaussian_sigma` to see which changes the grain mask least, ensuring quality over different scan sizes.                                                                                                                                                                                                                                                          |
```

Comment:
Good spot, this wouldn't have been caught by the `codespell` hook by virtue of the `_`.

Reply:
Thank you!